SPECIFICATION.md

```diff
@@ -122,19 +122,19 @@ If we had received the hash for tree index 5 before and verified it then we woul
 
 It should be noted that this allows us to have random access to any block in the feed with content verification in a single round trip
```

Comment:
"It should be noted that **this** allows..." should replace "**this**" with something more specific.  I wasn't sure what though.